torch/csrc/distributed/rpc/process_group_agent.h

```diff
@@ -12,6 +12,10 @@ namespace torch {
 namespace distributed {
 namespace rpc {
 
+constexpr int kDefaultNumSendRecvThreads = 4;
+constexpr std::chrono::milliseconds kDefaultRpcTimeout =
```

Comment:
Can we put this in RpcAgent.h since all agents should share the same default timeout?